cli/cmd/publicdomain/rm.go

```diff
@@ -0,0 +1,14 @@
+package publicdomain
+
+import (
+	"github.com/rancher/rio/cli/cmd/rm"
+	"github.com/rancher/rio/cli/pkg/clicontext"
+	"github.com/rancher/rio/types/client/space/v1beta1"
+)
+
+type Rm struct {
+}
+
+func (r *Rm) Run(ctx *clicontext.CLIContext) error {
+	return rm.Remove(ctx, client.PublicDomainType)
```

Comment:
Does remove with with the `domain name`, not just the resource Name?

Reply:
it removes resourceName. If we want to use generated name, we have to look up for domainName.